src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Legacy/AddTagHelperChunkGenerator.cs

```diff
@@ -29,11 +29,11 @@ internal class AddTagHelperChunkGenerator : SpanChunkGenerator
 
     public string LookupText { get; }
 
-    public string DirectiveText { get; set; }
+    public string DirectiveText { get; }
 
-    public string TypePattern { get; set; }
+    public string TypePattern { get; }
 
-    public string AssemblyName { get; set; }
+    public string AssemblyName { get; }
 
     public List<RazorDiagnostic> Diagnostics { get; }
```

Comment:
Is the expectation that this becomes `ImmutableArray<ReportDiagnostic>` in the future?

Reply:
My expectation is that this gets removed entirely at some point 🙂.